jeremy/index.html

```diff
@@ -12,4 +12,32 @@
         <div class="jeremy">hello world, Jeremy</div> 
     </body>
 </html>
- 
\ No newline at end of file
+<html>
+    <head>
+        <style>
+            #notes {
+                width: 50%;
+                height:300px;
+            }
+        </style>
+    </head>
+    <body>
+        <h1>Jeremy's notes</h1>
+        <div>
+            <textarea id='notes'></textarea>
+        </div>
+        <button id='save'>Save notes</button>
```

Comment:
I don't see a clear button here. Did you intend to add one? you can make changes to on the branch that you are already working on and push those changes which should be visible in the PR. Do you want to try?

Reply:
Oh, something must've gone wrong as the clear button was added in visual code studio and I can see the clear button on my website. I will try to push the changes up.